web/pdf_page_view.js

```diff
@@ -188,6 +191,8 @@ class PDFPageView extends BasePDFPageView {
     this.maxCanvasPixels =
       options.maxCanvasPixels ?? AppOptions.get("maxCanvasPixels");
     this.maxCanvasDim = options.maxCanvasDim || AppOptions.get("maxCanvasDim");
+    this.capCanvasAreaFactor =
+      options.capCanvasAreaFactor || AppOptions.get("capCanvasAreaFactor");
```

Comment:
Is `0` a valid value for the option/preference, because if so the fallback should probably be tweaked a little?
```suggestion
      options.capCanvasAreaFactor ?? AppOptions.get("capCanvasAreaFactor");
```